packages/opencensus-exporter-trace-zipkin/test/test-zipkin.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2018 OpenCensus Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {types} from '@opencensus/opencensus-core';
+import {classes} from '@opencensus/opencensus-core';
+import * as assert from 'assert';
+import * as http from 'http';
+import * as mocha from 'mocha';
+import * as nock from 'nock';
+
+import {ZipkinExporterOptions, ZipkinTraceExporter} from '../src/zipkin';
+
+/** Interface with request response model */
+interface RequestResponse {
+  statusCode: number;
+  statusMessage: string;
+}
+
+/** Default options for zipkin tests */
+const zipkinOptions = {
+  url: 'http://localhost:9411/api/v2/spans',
+  serviceName: 'opencensus-tests'
+} as ZipkinExporterOptions;
+
+/** Default config for traces tests */
+const defaultConfig: types.TracerConfig = {
+  samplingRate: 1
+};
+
+/** Run a nock server to replace zipkin service */
+const runNockServer = () => {
+  nock('http://localhost:9411/')
+      .persist()
+      .post('/api/v2/spans')
+      .reply(202)
+      .post('/api/v2/spanswrong')
+      .reply(404);
+};
+
+/**
+ * Checks if zipkin service is not running for tests
+ */
+const isNotZipkinRunning = (callback: Function) => {
```

Comment:
Instead of using the existence of a `Zipkin` server to alter test behavior, can we use an environmental variable like in https://github.com/census-instrumentation/opencensus-node/pull/18#discussion_r190952198

Reply:
Ok, created the environmental variable.